src/main/java/com/example/restservice/models/Question.java

```diff
@@ -0,0 +1,50 @@
+package com.example.restservice.models;
+import jakarta.persistence.*;
+import org.hibernate.mapping.List;
+import org.hibernate.mapping.Set;
```

Comment:
Shouldn't it be `import java.util.Set` ?
For me it seems like compilation will fail because  in further code you are using `Set` as  `java.util.Set` class, not as `hibernate.mapping.Set`.